client/components/user/user.html

```diff
@@ -21,8 +21,10 @@ <h2 class="well">{{userCtrl.user.name}}</h2>
 
         <div class="col-md-6">
              <posts-list posts="userCtrl.posts" 
+                         owner="userCtrl.user"
                          creator="userCtrl.currentUser" 
-                         owner="userCtrl.user">
+                         current-user="userCtrl.currentUser"
```

Comment:
Remove current-user property because creator property always would pass currentUser instance to component